core/src/main/scala/org/genivi/sota/core/CampaignResource.scala

```diff
@@ -82,6 +93,23 @@ class CampaignResource(namespaceExtractor: Directive1[AuthedNamespaceScope],
     complete(CampaignStats.get(id, updateService))
   }
 
+  def toggleDelta(id: Campaign.Id): Route = {
+    val sha256size = 64
+    parameters(('deltaFromName.as[PackageId.Name].?,
```

Comment:
These types of validations should not be done on the controller level. We should have a type for this hash (for example `Refined[_, _]` and use an extractor with `as` and let akka handle that for us.